### ISSUE TITLE
autodeploy.sh script doesn't cover macOS permissions

### ISSUE THREAD

**Issue by aksm** [esmero/archipelago-deployment#240]
# What?
The current autodeploy.sh script doesn't take into account that the solrcore and iiifcache folders on macOS need current user permissions.

**Comment by DiegoPino** [comment]
```sh
if [ "$(uname)" == "Linux" ]; then sudo chown -R 8183:8183 persistent/iiifcache &&
sudo chown -R 8983:8983 persistent/solrcore; fi
```
Something like that? `uname` gives you `Darwin` on any MacOS but assuming we always assume Linux subsystem in Windows too right? If not we we can compare !== "Darwin"

**Comment by aksm** [comment]
Resolved by 5c55cd317710cdaad5e8d96141227ba4ddb54ebe.
